packages/bot-web-ui/src/components/trading-view-chart/trading-view.tsx

```diff
@@ -5,7 +5,7 @@ const TradingViewComponent = () => {
         <iframe
             id='trading-view-iframe'
             style={{ width: '100%', height: '100%', backgroundColor: 'white' }}
-            src={'https://tradingview.deriv.com/deriv?hide_banner=1&hide-signup=true'}
+            src='https://charts.deriv.com/deriv?hide-signup=true'
```

Comment:
we don't need hide_banner anymore?

Reply:
the hide banner was not working...
we wanted to hide it on binary-bot but since we copied the code from binary-bot it came along with it.
So we need to show the banner 👍 
@balakrishna-deriv